pkgs/os-specific/linux/libsemanage/default.nix

```diff
@@ -6,24 +6,22 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "libsemanage-${version}";
-  version = "2.7";
+  version = "2.8";
   inherit (libsepol) se_release se_url;
 
   src = fetchurl {
     url = "${se_url}/${se_release}/libsemanage-${version}.tar.gz";
-    sha256 = "0xnlp1yg8b1aqc6kq3pss1i1nl06rfj4x4pyl5blasnf2ivlgs87";
+    sha256 = "1qm8pr67w08xrkqfky6qvy3r18w4bh873qr1dj960m0yqp9fh38w";
   };
 
   nativeBuildInputs = [ bison flex pkgconfig ];
   buildInputs = [ libsepol libselinux bzip2 audit ]
     ++ optionals enablePython [ swig python ];
 
   preBuild = ''
-    makeFlagsArray+=("PREFIX=$out")
+    makeFlagsArray+=("PREFIX=/")
```

Comment:
Why do this? Can this backfire on systems without sandboxing?

Reply:
The build files where changed in 2.8 so that `makeFlagsArray+=("DESTDIR=$out")` below is (correctly) prepended to `PREFIX`, thus with `PREFIX=$out` you get the same path, twice.

Reply:
Then we should actually set `PREFIX=$out` and `DESTDIR=/`. This is because PREFIX is the hardcoded path, DESTDIR is just an offset:
https://github.com/SELinuxProject/selinux/blob/ee8f7a870c625de139aa271eae0c40578488c2f6/libsemanage/src/Makefile#L95

Reply:
Might as well migrate to use an attribute:
```nix
{
  makeFlagsArray = [
    "DESTDIR=/"
    "PREFIX=${placeholder "out"}"
    "PYTHONLIBDIR=lib/${python.libPrefix}/site-packages"
  ];
}
```

Reply:
cc #65718

Reply:
Yup, I already did that in `libselinux`: https://github.com/NixOS/nixpkgs/commit/af9f765b1f5f4243a0b1b247e66de4849705225c#diff-025612052707a295d6f7424b3dba06bc It should be done here as well.